packages/showcases/css/stories/components/text-input/examples/default.html

```diff
@@ -1,10 +1,28 @@
 <div class="block">
   <label class="vtmn-text-input_label" for="my-text-input">Label</label>
-  <input
-    type="text"
-    class="vtmn-text-input"
-    id="my-text-input"
-    placeholder="Placeholder Text"
-  />
+  <div>
+    <input
+      type="text"
+      class="vtmn-text-input"
+      id="my-text-input"
+      placeholder="Placeholder Text"
+    />
+    <span>
+      <svg
```

Comment:
For this example, it could be better to use another icon than calendar.
Also, it could be better to say that consumer needs to install `@vtmn/icons` and display "icon font" example with `vtmx-***` CSS classes inside a `<span>` no?